include/kllvm/binary/ProofTraceParser.h

```diff
@@ -0,0 +1,626 @@
+#ifndef PROOF_TRACE_VALIDATOR_H
+#define PROOF_TRACE_VALIDATOR_H
+
+#include <kllvm/ast/AST.h>
+#include <kllvm/binary/deserializer.h>
+
+#include <fmt/format.h>
+
+#include <iostream>
+#include <string>
+
+namespace kllvm {
+
+namespace detail {
+
+constexpr uint64_t word(uint8_t b) {
+  uint64_t w = uint64_t(b);
+  for (auto i = 0; i < 7; ++i) {
+    w = (w << 8u) | w;
+  }
+  return w;
+}
```

Comment:
Can we unify these two implementations? https://github.com/runtimeverification/llvm-backend/blob/e8c7319292ad82f7ca93e0faa4aa99656104cc17/lib/codegen/ProofEvent.cpp#L25-L32

Reply:
good catch! I pushed a commit to unify them.